package.json

```diff
@@ -8,7 +8,8 @@
     "build": "tsc && vite build",
     "preview": "vite preview",
     "prepare": "husky install",
-    "test": "jest",
+    "test": "vitest",
```

Comment:
no need for vitest

Reply:
We are not goingo to accept any other test configurations please check and work whit the current config